src/js/src/components/SearchBox.vue

```diff
@@ -152,6 +154,9 @@ export default {
       updateShowModal:'updateShowModal',
       updateCurrentModal:'updateCurrentModal'
     }),
+    checkKeyPresses() {
+      !event.shiftKey ? (event.preventDefault(),this.launchNewSearch()) : null
+    },
```

Comment:
I guess you utilised the natural power of textarea to support line breaks but the threw a 'shift button check' in there as the enter button is already hijacked for submit - very clever.

I now why we changed the input to textarea and all is fine with this app but in other applications we would have to thread carefully with this approach is this can wreak havoc on accessibility.